src/todo-list-core/todo-list-core.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
class libraries using .NET Standard, API is still .NET Framework instead of .NET Core because using Web API